src/playback/internal/playbackconfiguration.h

```diff
@@ -47,6 +47,8 @@ class PlaybackConfiguration : public IPlaybackConfiguration, public muse::async:
     bool playHarmonyWhenEditing() const override;
     void setPlayHarmonyWhenEditing(bool value) override;
 
+    muse::async::Notification playNotesWhenEditingChanged() const override;
```

Comment:
Sorry for nitpicking, but it would be preferable to keep this close to`bool playNotesWhenEditing()`